processing/src/main/java/io/druid/segment/DimensionHandlerUtil.java

```diff
@@ -35,6 +36,11 @@ public static DimensionHandler getHandlerFromCapabilities(
   )
   {
     DimensionHandler handler = null;
+
+    if (dimensionName.equals(Column.TIME_COLUMN_NAME)) {
```

Comment:
can we add a reminder that this logic probably needs to change with new dim typing work?